content/settings/footer.md

```diff
@@ -2,11 +2,17 @@
 project:
   name: Code for Africa
   url: https://codeforafrica.org
-  description:
-    This site is a project of Code for Africa, the continent’s largest
-    network of civic technology and data journalism labs. All content is
-    released under a Creative Commons 4 Attribution Licence. Reuse it to help
-    empower your own community.
+  description: >-
+    This site is a project of Code for Africa, the continent’s largest network
+    of civic technology and data journalism labs. All content is released under
+    a Creative Commons 4 Attribution Licence. Reuse it to help empower your own
+    community.
+
+
+    a
+
+
+    vbdd
```

Comment:
This reminds me of Donald Trumps' [Covfefe](https://en.wikipedia.org/wiki/Covfefe)